app/models/LiabilityCalculation/LiabilityCalculationResponse.scala

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.LiabilityCalculation
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+import models.LiabilityCalculation.taxCalculation.TaxCalculation
+import models.LiabilityCalculation.taxDeductedAtSource.TaxDeductedAtSource
+
+sealed trait LiabilityCalculationResponseModel
+
+case class LiabilityCalculationError(status: Int, message: String) extends LiabilityCalculationResponseModel
```

Comment:
You might consider moving case classes and their companion objects into their own scala files.